src/github_utils.py

```diff
@@ -0,0 +1,132 @@
+import json
+import os
+import requests
+from github import Github
+
+def get_pull_request_diff():
+    """Fetch the pull request diff content from GitHub"""
+    g = Github(os.getenv('GIT_TOKEN'))
+
+    # Parse GitHub event context
+    event_path = os.getenv('GITHUB_EVENT_PATH')
+    with open(event_path, 'r') as f:
+        event_data = json.load(f)
+
```

Comment:
Consider adding error handling for cases where the GITHUB_EVENT_PATH environment variable is not set, or the file does not exist.

```suggestion
|
```try:
event_path = os.getenv('GITHUB_EVENT_PATH')
with open(event_path, 'r') as f:
event_data = json.load(f)
except (OSError, IOError) as e:
print(f"Error reading GITHUB_EVENT_PATH: {e}")
return None #Or appropriate error handling